scripts/acceptUsers.js

```diff
@@ -1,20 +0,0 @@
-require('dotenv').load();
```

Comment:
Why are we getting rid of the four js script files?

Reply:
We never really used them anyway. They aren't actually part of the application.

Reply:
Gotcha, just for testing and stuff I guess? Maybe let's keep them in just in case?